### ISSUE TITLE
Multiline export in index.dart are not correctly parsed

### ISSUE THREAD

**Issue by voltaire-lre** [FlutterFlow/VS-Code-Extension#1]
When I pull the code (or if I start a code editing session) all import which takes more than a single line are deleted by the extension.

Ex : 
```
export 'get_calendar_event_from_google_calendar.dart'
    show getCalendarEventFromGoogleCalendar;
export 'delete_file_from_bucket.dart' show deleteFileFromBucket;
```

Become 
`export 'delete_file_from_bucket.dart' show deleteFileFromBucket;`

I just looked at the code and in the UpdateManager.ts the function parseIndexFile (line 537) is not performing correctly.
It splits by '\n' and therefore breaks the multiline import.

```
/**
 * Parses Dart index files
 * @param content Content of index file
 * @returns Map of exports
 */
function parseIndexFile(content: string): Map<string, string[]> {
  const index = new Map<string, string[]>();
  const lines = content.split('\n');
  for (const line of lines) {
    const match = line.match(/export '([^']+)' show (.+);/);
    if (match) {
      const filename = match[1];
      const exports = match[2].split(',').map(s => s.trim());
      index.set(filename, exports);
    }
  }
  return index;
}
```

**Comment by voltaire-lre** [comment]
I tried to modify the source code of the extension with the following code : 
```
/**
 * Parses Dart index files
 * @param content Content of index file
 * @returns Map of exports
 */
function parseIndexFile(content: string): Map<string, string[]> {
  const index = new Map<string, string[]>();
  const lines = content.split('\n');
  var i = 0;
  for (const line of lines) {
    const match = line.match(/export '([^']+)' show (.+);/);
    const partialMatchExport = line.match(/export '([^']+)'/);

    if (match) {
      const filename = match[1];
      const exports = match[2].split(',').map(s => s.trim());
      index.set(filename, exports);
    }
    else if (partialMatchExport && i < lines.length - 1) {
      const nextLine = lines[i+1];
      const filenamePartial = partialMatchExport[1];
      const partialMatchShow = nextLine.match(/\s*show (.+);/);
      if (partialMatchShow) {
        const nextLineExports = partialMatchShow[1].split(',');//.map(s => s.trim());
        index.set(filenamePartial, nextLineExports);
      }
    }
    i++;
  }
  return index;
}
```

It works correctly with exports written on two lines.

**Comment by ryandiazFF** [comment]
Thank you for filing this issue. Fixed by #2 . The fix is included in the just released extension version  1.1.3
